HomeTask2/HomeTask2.js

```diff
@@ -1,88 +1,230 @@
 
-function openpopup(value) {
+window.onload = function () {
```

Comment:
form meaningful functions. Segregate the functionality to one one function instead of clubbing everything into one.